aten/src/ATen/native/cuda/AdaptiveAveragePooling.cu

```diff
@@ -727,9 +727,9 @@ namespace {
 } // namespace
 
   Tensor& adaptive_avg_pool2d_out_cuda(
-    Tensor& output,
     const Tensor& input,
-    IntArrayRef output_size)
+    IntArrayRef output_size,
+    Tensor& output)
```

Comment:
Forgot the include your previous fix in native_functions.yaml? Can we make this as a separate PR since you have mastered ghstack? It will make the review and triage easier if anything goes wrong.

Reply:
@desertfire https://github.com/pytorch/pytorch/pull/61851